res/css/_common.pcss

```diff
@@ -481,13 +481,43 @@ legend {
     height: unset;
 }
 
-.mx_Dialog_cancelButton {
-    @mixin customisedCancelButton;
-    width: 18px;
-    height: 18px;
+.mx_Dialog_headerWithCancel div.mx_Dialog_cancelButton {
+  width: 30px;
+  height: 30px;
+  transform: rotate(45deg);
+  position: absolute;
+  top: 10px;
+  right: 0; 
+  
+  &.mx_Dialog_cancelButton {
+    width: 30px;
+    height: 30px;
+    &:after {
+      width: 26px;
+      height: 4px;
+      left: 2px;
+      top: 13px;
+    }
+    &:before {
+      width: 4px;
+      height: 26px;
+      top: 2px;
+      left: 13px;
+    }
+  }
+  
+  -webkit-border-radius: 2px;
+  -moz-border-radius: 2px;
```

Comment:
Declarations with prefixes are generated automatically when compiled, so you would not need to add ones manually.

Reply:
In fact, because your support browser are the 2 latest versions, you may drop to generate.